src/InBrain.tsx

```diff
@@ -86,10 +86,18 @@ const setOnSurveysCloseLister = (callback: (result: OnCloseSurveysData) => void
 const checkSurveysAvailable = () => wrapPromise<boolean>(() => InBrainSurveys.checkSurveysAvailable())
 
 /**
- * Show the Survey Wall. All the configs should be done `BEFORE` calling `showSurveys()`.
+ * @deprecated Use openWall() instead
  */
 const showSurveys = () => wrapPromise<void>(() => InBrainSurveys.showSurveys());
```

Comment:
1) call the new `openWall` method inside the `showSurveys`, i.e.
`const showSurveys = () => wrapPromise<void>(() => openWall(InBrainWallOption.all));`

2) move the method bellow the actual methods, i.e. add the new //--- Deprecated -- section above the `Unsupported`